sklearn/feature_selection/rfe.py

```diff
@@ -65,6 +66,15 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         If within (0.0, 1.0), then ``step`` corresponds to the percentage
         (rounded down) of features to remove at each iteration.
 
+    step_one_threshold : int or None (default=None)
+        If specified step int or float equates to > 1 feature then the
+        threshold number of remaining features when to revert to step = 1
+
+    step_decay : boolean, optional (default=False)
+        If step is a float whether to calculate the percentage of features
```

Comment:
How about "If step is a float, whether to remove a ``step`` fraction of the original number of features, or a ``step`` fraction of the current number of features in each step". Still not very clear though :-/
"Whether to remove a fraction of the original number feature or current number of features at each step?"